saga-sdk-java/src/main/java/games/mythical/saga/sdk/client/SagaItemClient.java

```diff
@@ -142,11 +143,12 @@ public String transferItem(String inventoryId, String destOauthId) throws SagaEx
         }
     }
 
-    public String transferItemBulk(String idempotencyId, String destOauthId, List<String> inventoryIds) throws SagaException {
+    public String transferItemBulk(String idempotencyId, String destOauthId, List<String> inventoryIds, Boolean prefundGas) throws SagaException {
```

Comment:
This isn't needed cuz they will always have to prefund the gas for bulk transfer since its for a guest account

Reply:
ok i can hardcode it in gateway

Reply:
It's already hardcoded to true in there, at least it should be.